lib/teiserver/game/servers/balancer_server.ex

```diff
@@ -213,17 +213,14 @@ defmodule Teiserver.Game.BalancerServer do
           |> Enum.map(fn userid ->
             %{
               userid =>
-                BalanceLib.get_user_balance_rating_value(userid, rating_type)
-                |> fuzz_rating(opts[:fuzz_multiplier])
+                BalanceLib.get_user_rating_rank(userid, rating_type, opts[:fuzz_multiplier])
             }
```

Comment:
ah yes indeed, you're correct, it's not great. But I think this change is big enough already so let's leave it this way. Perhaps add a TODO mentioning that it would be better to grab everything in as few query as possible?